sys/zuse.hoon

```diff
@@ -678,11 +678,16 @@
     $%  {$bel ~}                                        ::  make a noise
```

Comment:
This says `old blit`. Theres `++ dill-blit :: new blit` further down. Do you know what the deal is with that? This seems like a good time to unify those, or clean them up, whichever's desired.

`/mar/bill/blit` exists, and it uses `++dill-blit`, so maybe that's where these changes should go, and have `++blit` be removed? Using `++dill-blit` exclusives, like `%url`, crashes with abort.

Reply:
Actually, that second paragraph is incorrect issue analysis. `%url` appears in both. But it crashes nonetheless!